qutebrowser/config/configdata.py

```diff
@@ -321,6 +321,10 @@ def data(readonly=False):
              SettingValue(typ.String(none_ok=True), 'en-US,en'),
              "Value to send in the `accept-language` header."),
 
+            ('referer-header',
+             SettingValue(typ.Referer(), 'same-domain'),
+             "Send the Referer header"),
```

Comment:
I originally spelled it correct in some placed, but then settled on being consistently incorrect (which is actually correct)... :-/